src/spicedb-common/components/relationshipeditor/RelationshipEditor.tsx

```diff
@@ -936,14 +934,14 @@ export function RelationshipEditor(props: RelationshipEditorProps) {
     });
   };
 
-  const { drawCell, provideEditor } = useCustomCells(
```

Comment:
This is one of the things that changed in v6 - you provide `customRenderers` directly instead of wrapping in `drawCell` and `provideEditor`.